src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.scss

```diff
@@ -18,3 +18,9 @@
 .stats-table {
   min-width: 800px;
 }
+
+.btn{
+  background-color: #007BFF;
+  color: white;
+  margin-left: 5px;
+}
```

Comment:
Let's try to use the UI components/styles (i.e., CSS classes) provided by bootstrap.